docs/examples/devicetypeprovider/implementation.js

```diff
@@ -0,0 +1,112 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Box, SideNavigation, DeviceTypeProvider, Button, Flex } from 'gestalt';
+
+export default function Example(): Node {
+  const [deviceType, setDeviceType] = React.useState('desktop');
+
+  return (
+    <Flex>
+      <Box padding={2}>
```

Comment:
I wanted to add spacing around the Button so it's not touching the edges of the container. With Flex it would add space between items, not with parent